acceptance/api/v1/configurations_mutation_test.go

```diff
@@ -569,6 +573,8 @@ var _ = Describe("Configurations API Application Endpoints, Mutations", func() {
 			json.Unmarshal(bodyBytes, &responseBody)
 			Expect(responseBody["errors"][0].Title).To(
 				Equal("application 'bogus' does not exist"))
+
+			env.CleanupConfiguration(configuration)
```

Comment:
`defer` this?

Reply:
Done. e5133a021247089698f52f839b0ef9936ec8390e